packages/worker/src/sdk/auth/auth.ts

```diff
@@ -59,7 +59,7 @@ export const reset = async (email: string) => {
 
   // exit if user has sso
   if (await userSdk.isPreventPasswordActions(user)) {
-    throw new HTTPError("Password reset is disabled for this user", 400)
+    return
```

Comment:
Will this not hide some unexpected behaviour?

Reply:
It'll result in requesting a reset email to succeed, effectively means "if you have a valid account you will receive an email". In reality the email will never be sent as we exit early here. Same as above when we exit if the user is not found